inventory/technical/reporting.rst

```diff
@@ -0,0 +1,191 @@
+==========================
+Inventory: Under the wood
+==========================
+
+The inventory valuation example
+********************************
+
+Let's take a look at the inventory valuation report.
+The report can be found in (Inventory -> Reporting -> Inventory Valuation)
+A popup appears asking if you want to compute the current inventory or the inventory at a specific date.
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#action=912&active_id=1&model=product.product&view_type=list&menu_id=
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#active_id=2&model=product.product&view_type=list&menu_id=
```

Comment:
links stops at the '?' in the rich diff, not a real issue but forces to select and ctrl + C/V

Reply:
solved